test/unit-test/core_pkcs11_mbedtls_utest.c

```diff
@@ -1481,11 +1477,11 @@ void test_pkcs11_C_CreateObjectECPubKeyBadAtt( void )
     CK_BBOOL xTrue = CK_TRUE;
     CK_BBOOL xFalse = CK_FALSE;
     char * pucPubLabel = pkcs11configLABEL_DEVICE_PUBLIC_KEY_FOR_TLS;
-    size_t xLength = 256;
+    size_t xLength = 67;
```

Comment:
what are these random looking magic numbers?
 it would be better to add a meaningful define

Reply:
Swapped  to use a macro that represents this data